guides/hack/08-arrays-and-collections/01-introduction.md

```diff
@@ -57,6 +57,8 @@ Vec\map(keyset[1, 2], $x ==> $x + 1); // vec[2, 3]
 | Type Refinement                              | `$v is vec<_>`               | `$d is dict<_, _>`             | `$k is keyset<_>`              |
 | `Awaitable` Consolidation                    | `Vec\from_async($v)`         | `Dict\from_async($d)`          | `Keyset\from_async($x)`        |
 
+* `$container` can be a Hack Array or Hack Collection
```

Comment:
you probably want backticks around the asterix - this is currently parsed as a single-item unordered list (`<ul><li>foo</li></ul>`)